### PR TITLE
Update Auth Success Page

### PR DESCRIPTION
**Pull request by alxndr13**
Updated the Auth Success Page, delivering the page using an embed filesystem and added a "powered by inovex" part.

Now looks like this:

image

### CODE REVIEW

File: internal/auth/assets/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>CalendarSync</title>
+</head>
+<body style='font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;'>
+    <div style="text-align: center; padding-top: 30px;">
+        <h2 style="color:#051c59; font-weight: 500; font-size: 30px; margin-bottom: 10px;">CalendarSync authentication successful!</h2>
+        <p style="font-size:20px; color:#7df481; margin-top: 10px;">You can now close this window.</p>
```
Comment (MichaelEischer) [inline internal/auth/assets/index.html:9]: While I like green in general, that green is a bit too bright in my opinion.

Comment (alxndr13) [reply internal/auth/assets/index.html:9]: That's what a colleague in the office today said as well. 

For the context: it's the "accent" color of the inovex color palette. 

I'll see what other options we have! 👀

Comment (alxndr13) [reply internal/auth/assets/index.html]: image

Added another accent color, from the "blue color accent" palette. WDYT?

Comment (MichaelEischer) [reply internal/auth/assets/index.html]: Much better :-)

**Review by MichaelEischer — APPROVED** [review]

**Pull request merged**